src/evagram_input/input_tool/input_tool.py

```diff
@@ -1,26 +1,36 @@
+from . import config
+from . import test_config
 import pickle
 import os
 import psycopg2
-from dotenv import load_dotenv
-
-load_dotenv()
 
 
 class Session(object):
     """The session object opens a connection to the evagram database and serves as the
     central point for inputting data from a swell task."""
-    def __init__(self, owner: str, experiment: str, eva_directory: str):
+    def __init__(self, owner: str, experiment: str, eva_directory: str, test_local=False):
+        self.username = config.user
+        self.dbname = config.dbname
+        self.host = config.host
+
+        if test_local:
+            self.username = test_config.user
+            self.dbname = test_config.dbname
+            self.host = test_config.host
```

Comment:
This is pretty much what I described in our last meeting, but here's an alternative solution:

Store `user/database/host` information in a `dbconfig` class defined in its own separate `dbconfig` module.  Move config.py and test_config.py into that module.  Initialize the object with the `test_local` boolean to pull in the proper db parameters.  Then have a `get` method that returns a dictionary like `{'user':self.user, 'database':self.database, 'host':self.host'}`.  In the session class where psycopg2 is called, we can do `connect(**dbparams)`.  Think that could work?  It seems like a cleaner way to organize concerns.

Reply:
I believe that could work, I like the idea of moving the config files outside the session class. How will the `dbconfig` object interact with the session class? Will this also mean `dbconfig` is another dependency in evagram for testing?

Reply:
I was thinking a bit more about the implementation, and I thought that I could replace the `test_local` boolean with an environment variable `APP_ENVIRONMENT` which I can set to the current state of the project (local). This way the `dbconfig` module can configure the database in multiple ways based on the value in `APP_ENVIRONMENT`. @tariqhamzeyssai Let me know if that sounds good with you.